io/src/test/scala/fs2/io/TestUtil.scala

```diff
@@ -0,0 +1,51 @@
+package fs2.io
+
+import java.lang.Thread.UncaughtExceptionHandler
+import java.net.InetSocketAddress
+import java.nio.channels.AsynchronousChannelGroup
+import java.nio.channels.spi.AsynchronousChannelProvider
+import java.util.concurrent.ThreadFactory
+import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.locks.ReentrantLock
+
+import org.scalacheck.Prop
+
+
+object TestUtil {
+
+  def namedACG(name:String):AsynchronousChannelGroup = {
+    val idx = new AtomicInteger(0)
+    AsynchronousChannelProvider.provider().openAsynchronousChannelGroup(
+      8
+      , new ThreadFactory {
+        def newThread(r: Runnable): Thread = {
+          val t = new Thread(r, s"fs2-AG-$name-${idx.incrementAndGet() }")
+          t.setDaemon(true)
+          t.setUncaughtExceptionHandler(new UncaughtExceptionHandler {
+            def uncaughtException(t: Thread, e: Throwable): Unit = {
+              println("-------------- UNHANDLED EXCEPTION ------------")
+              e.printStackTrace()
+            }
+          })
+          t
+        }
+
+      }
+    )
+  }
+
+
+
+  def localAddress(port: Int) = new InetSocketAddress("localhost", port)
+  val localBindAddress = localAddress(9999)
+
+
+  // the io specs may fight for local machine resources
```

Comment:
Ok I can remove it. I just want to be sure we really won't get any of the two tests to run concurrently hence we share the bind port.